### ISSUE TITLE
[Accessibility] [Semantics] Muda’s button is identified as a img

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#861]
**Describe the bug**
The "Muda logo" button is identified as an image with no alt attribute. As behaviour, it should role as button for a better accessible user experience.
Ensure <img> elements have alternate text or a role of none or presentation
more informationLink opens in a new window
Element Location:
.h-10<img class="h-10" src="https://cambiatus-uploads.s3.us-east-1.amazonaws.com/cambiatus-uploads/93a2409d-ec8e-4b6d-a98c-f47544ddfe52">
To solve this problem, you need to fix at least (1) of the following:
Element does not have an alt attribute
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"
Found:Automatically
Impact:critical
cat.text-alternatives
wcag2a
wcag111
section508
section508.22.a
TTv5
TT7.a
TT7.b
EN-301-549
EN-9.1.1.1
ACT
Found on:18/10/2024 at 12:18 PM

**To Reproduce**
1. Go to '(https://muda.cambiatus.io/join?redirect=%2Fdashboard)'
2. Run AxeDev Tools
3. See error

**Expected behavior**
It should be a button with a label like “muda’s home page” or something like it. The screenreader user doesn’t know what is that image purpose.

**Screenshots**

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**
 - OS:Windows
 - Browser Chrome
 - Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**

**Comment by ceciliabelem** [comment]
For mapping all defect statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
